demos/slots-stream/real-time-user-segmentation.ipynb

```diff
@@ -217,7 +217,23 @@
     "# Delete the stream if it exists\n",
     "client.delete(backend=\"stream\", table=TABLE, if_missing=fpb.IGNORE)\n",
     "# Create a new stream\n",
-    "client.create(backend=\"stream\", table=TABLE, shards=SHARDS_COUNT, retention_hours=DURATION_HOURS)"
+    "client.create(backend=\"stream\", table=TABLE, shards=SHARDS_COUNT, retention_hours=DURATION_HOURS)\n",
+    "\n",
+    "url = 'http://v3io-webapi:8081/users/' + os.getenv('V3IO_USERNAME') + '/examples/' + STREAM + '/'\n",
+    "headers = {\n",
+    "            \"Content-Type\": \"application/json\",\n",
+    "            \"X-v3io-function\": \"PutRecords\",\n",
+    "            \"X-v3io-session-key\": os.getenv('V3IO_ACCESS_KEY')\n",
+    "          }\n",
+    "\n",
+    "records = []\n",
+    "\n",
+    "def send_payload(records):\n",
+    "    if (len(records) > 0):\n",
+    "        payload = {\n",
+    "            \"Records\": records\n",
+    "        }\n",
+    "        requests.post(url, json=payload, headers=headers, verify=False)"
```

Comment:
1. The web-API `headers` and `records` initialization code and the definition of the `send_payload` function shouldn't be part of the "Create a Stream" section. I would suggest the following: in the "Write Data to the Stream" section, change "Generate Timestamps" to "Initialization", change the anchor ID to `user-seg-demo-write-to-stream-init`, remove the text at the start of the doc cell, and copy the web-API initialization code to the code cell, before the timestamps generation code; (ideally, I would have used separate code cells, but that would require updating the NB's output counts, so let's avoid this). Remember to also edit the link to the renamed "Generate Timestamps" section from the start of the "Write Data to the Stream" doc cell.
2. PEP8 requires two empty lines before a `def` statement.
3. Consider splitting the `url` assignment line into two lines to conform to the PEP8 line-length restriction; the following passes PEP8 validation:
    ```python
    url = 'http://v3io-webapi:8081/users/' + os.getenv('V3IO_USERNAME') + \
        '/examples/' + STREAM + '/'
    ```

Reply:
Yes, the output remains the same with the PutRecords loop.

I will address your second comment about refactoring the notebook now.